ircbot/plugin/turing.py

```diff
@@ -0,0 +1,95 @@
+"""Is create turing complete?"""
+import re
+
+import markovify
+
+from ircbot import db
+
+final_model = None
+
+
+def register(bot):
+    bot.listen(r'turing', markov, flags=re.IGNORECASE, require_mention=True)
+    bot.listen(r'^!genmodels', build_models)
+
+    build_models(bot)
+
+
+def markov(bot, msg):
+    """Return the best quote ever"""
+    if final_model:
+        output = final_model.make_sentence(tries=200)
+        if output:
+            # Put a zero width space in every word to prevent pings
+            # This is also much simpler than using crazy IRC nick regex.
+            # Put it in the middle of the word since nicks are quoted
+            # using "<@keur>" syntax.
+            msg.respond(
+                ' '.join([w[:len(w) // 2] + '\u2060' + w[len(w) // 2:] for w in output.split()]),
```

Comment:
```suggestion
                mid = len(w) // 2
                ' '.join(w[:mid] + '\u2060' + w[mid:] for w in output.split()),
```

you can replace the list comprehension with a generator